### PR TITLE
Fixed jquery and made colleges autofill for signup 

### PR DESCRIPTION
**Pull request by christinachance**

### CODE REVIEW

**Review by rivera-fernando — APPROVED** [review]

**Review by Tcintra — APPROVED** [review]

**Review by rivera-fernando — APPROVED** [review]

File: project/src/main/java/com/google/sps/classes/EmailSender.java [hunk 1]
```diff
@@ -49,13 +49,20 @@ protected PasswordAuthentication getPasswordAuthentication() {
...
         StringBuilder htmlBuilder = new StringBuilder();
-        htmlBuilder.append("<p>");
-        htmlBuilder.append("An account on Covidiena has been created with this email. To verify this is you, please click ");
-        htmlBuilder.append("<a href='covidiena.appspot.com/login.html'>here</a>.");
-        htmlBuilder.append("</p>");
+        if (password.equals("")){
+            message.setSubject("[Covidiena] Account Verification");
+            htmlBuilder.append("<p>");
+            htmlBuilder.append("An account on Covidiena has been created with this email. To verify this is you, please click ");
+            htmlBuilder.append(" <a href='covidiena.appspot.com/login.html'>here</a>.");
```
Comment (wulian1223) [inline project/src/main/java/com/google/sps/classes/EmailSender.java:57]: remove space before <a>

File: project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        for(Entity entity:results.asIterable()){
+            if(email.equals(((String)entity.getProperty("email")).toLowerCase())){
+                Entity user = null;
+                long userId = entity.getKey().getId();
+                try {
+                    Key userKey = KeyFactory.createKey("User", userId);
+                    user = datastore.get(userKey);
+                }
+                catch ( Exception EntityNotFoundException){
+                    return;
```
Comment (wulian1223) [inline project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java:63]: log something for debug purpose.

File: project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+                datastore.put(user);
+
+                EmailSender.sendEmail(email, password);
+                session.setAttribute("accountExists", true);
+                response.sendRedirect("/login.html");
+                return;
+            }
+        }
+        session.setAttribute("accountExists", false);
```
Comment (wulian1223) [inline project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java]: remove duplicate setting.

File: project/src/main/java/com/google/sps/classes/EmailSender.java (hunk 1 above)
Comment (christinachance) [reply project/src/main/java/com/google/sps/classes/EmailSender.java:57]: done.

File: project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java (hunk 2 above)
Comment (christinachance) [reply project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java:63]: done.

File: project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java (hunk 3 above)
Comment (christinachance) [reply project/src/main/java/com/google/sps/servlets/SearchAccountServlet.java]: done.

**Review by wulian1223 — APPROVED** [review]
LGTM after fixing comments

**Pull request merged**